.github/workflows/update-nix.yml

```diff
@@ -0,0 +1,30 @@
+name: Update niv
+
+on:
+  schedule:
+    # every day at 8am UTC
+    - cron: '0 8 * * *'
+
+jobs:
+  update-dependencies:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2.3.3
+    - uses: cachix/install-nix-action@v11
+    # https://github.com/nmattia/niv/issues/280
+    - run: GITHUB_PATH= nix-shell --run "niv update"
+    # TODO: Up to the authors of Waypoint to set up a binary cache at cachix.org
+    # - uses: cachix/cachix-action@v6
```

Comment:
Could uncomment this to populate a shared binary cache at  waypoint.cachix.org. Highly recommended for onboarding other collegues as it reduces initial time for `nix-shell` to load